provisioner/clusterpy.go

```diff
@@ -295,11 +295,15 @@ func (p *clusterpyProvisioner) Provision(ctx context.Context, logger *log.Entry,
 	// accounts.
 	bucketName := fmt.Sprintf(clmCFBucketPattern, strings.TrimPrefix(cluster.InfrastructureAccount, "aws:"), cluster.Region)
 
-	err = createOrUpdateClusterStack(ctx, channelConfig, cluster, values, awsAdapter, bucketName)
+	outputs, err := createOrUpdateClusterStack(ctx, channelConfig, cluster, values, awsAdapter, bucketName)
 	if err != nil {
 		return err
 	}
 
+	for _, o := range outputs {
+		values[aws.StringValue(o.OutputKey)] = aws.StringValue(o.OutputValue)
```

Comment:
Let's either have a prefix or put the outputs as a nested map (i.e. `values["ClusterStackOutputs"] = {…}`). We don't want to accidentally overwrite something important.

Reply:
They shouldn't clash as the naming scheme is different.

Reply:
Still, I'd group the stack outputs under one value (or with a prefix, but that's a bit worse in my opinion). It'll also be easier to figure out where something is coming from in the templates.

Reply:
Done, nested stack outputs under `values["ClusterStackOutputs"]`.